### ISSUE TITLE
Text render error on Firebase Functions

### ISSUE THREAD

**Issue by redpandatronicsuk** [chearon/node-canvas-prebuilt#17]
I am using node-canvas-prebuilt in Firebase Functions. I have an issue with rendering text, all characters are rendered as ☐
<img width="308" alt="screen shot 2017-07-23 at 14 06 05" src="https://user-images.githubusercontent.com/8609582/28499534-1b44b208-6fb0-11e7-9109-68efa9c8bcf2.png">

I thought it might be an issue with the installed font, but I see the same error when loading a bundled ttf file.

Version 1.6.0 is the latest one I can use, for later versions I get the `libgobject-2.0.so.0: undefined symbol: g_option_group_unref` compile time error I have seen here already mentioned in another issue.

Firebase functions are run inside these docker containers: https://github.com/GoogleCloudPlatform/nodejs-docker/blob/master/runtime-image/Dockerfile

**Comment by chearon** [comment]
That happens when Pango can't load a font module. I just tried running some font code with that docker image and here was the output:

```
(process:28): Pango-CRITICAL **: No modules found:
No builtin or dynamically loaded modules were found.
PangoFc will not work correctly.
This probably means there was an error in the creation of:
  '/etc/pango/pango.modules'
You should create this file by running:
  pango-querymodules > '/etc/pango/pango.modules'

(process:28): Pango-WARNING **: failed to choose a font, expect ugly output. engine-type='PangoRenderFc', script='latin'

(process:28): Pango-WARNING **: failed to choose a font, expect ugly output. engine-type='PangoRenderFc', script='common'
```

The problem is that neither the old builds (1.4.0, 1.5.0, 1.6.0) or the new builds (1.6.5, 2.0.0 alpha 1 and 2) are going to work for you. The old builds need to run on a system that has Pango "modules" (your docker image doesn't). The new build doesn't have this restriction but needs a certain GLib that many old systems don't have.

The docker image has GLib 2.42 but I'm distributing something that references functions added in GLib 2.60.

Basically everyone is having the same problems and I need to figure out how to distribute GLib or something that doesn't depend on it

**Comment by redpandatronicsuk** [comment]
Thanks for your reply. I'll ask the Firebase guys if they can update the docker image. In the meanwhile, I found a workaround solution using https://github.com/kaivi/node-canvas-text to draw the text.
